raspyrfm_client/device/manufacturer/hama/_121938.py

```diff
@@ -0,0 +1,47 @@
+from raspyrfm_client.device import actions
+from raspyrfm_client.device.base import Device
+
+
+class HAMA(Device):
```

Comment:
Oh okay. I was assuming this because it only implemented get_pulse_data() which is not defined in the Device base class. If this is a necessary method for all devices it should be declared in the base class.